cfpb/design-manual
# Fix the skip to content link

The link was once messing up the layout, but had been hidden entirely at
some point. This change duplicates the skip link styles from
cfgov-refresh.

Fixes #270

## Additions

- Adds skip link styles from cfgov-refresh.

## Testing

- Run `grunt build`
- Navigate to `http://localhost:4000/` and press `tab`, you should see the button

[Preview this PR without the whitespace changes](?w=0)

## Screenshots

<img width="1011" alt="screen shot 2018-07-02 at 4 53 36 pm" src="https://user-images.githubusercontent.com/1280430/42188175-7421d8e8-7e18-11e8-8997-6906383359ae.png">

## Checklist
